specification/applicationinsights/resource-manager/microsoft.insights/stable/2015-05-01/componentAnnotations_API.json

```diff
@@ -0,0 +1,344 @@
+            }
+          },
+          "400": {
+            "description": "Invalid time",
+			"schema": {
+              "$ref": "#/definitions/AnnotationCreateError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AnnotationsCreate": {
+            "$ref": "./examples/AnnotationsCreate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/components/{resourceName}/Annotations/{annotationId}": {
+      "delete": {
+        "description": "Delete an Annotation of an Application Insights component.",
+        "operationId": "Annotations_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AnnotationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The annotation that was successfully deleted.",
+            "schema": {              
+			  }
+          }
+        },
+        "x-ms-examples": {
+          "AnnotationsDelete": {
+            "$ref": "./examples/AnnotationsDelete.json"
+          }
+        }
+      },
+      "get": {
+        "description": "Get the annotation for given id.",
+        "operationId": "Annotations_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AnnotationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Annotation object for this id.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Annotation"
+              }
+		    }
+          },
+          "404": {
+            "description": "The Annotation with given id does not exist.",
+            "schema": {
+              "$ref": "#/definitions/AnnotationCreateError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AnnotationsGet": {
+            "$ref": "./examples/AnnotationsGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Annotation": {
+      "type": "object",
+      "description": "Annotation associated with an application insights resource.",
+      "properties": {
+        "AnnotationName": {
```

Comment:
all RP specific properties should go inside the "properties" property. Top level allowed properties are id, name, type, tags, location etc.

Reply:
So properties like category, eventtime etc need to be moved.

Reply:
This is a released version in PROD for 2+ years now. We'll update this in next api-version.

Reply:
Ok, since this is existing behavior which will break clients, lets identify the path forward in the next major version update.